SnakeLadderApplication/src/services/SnakeLadderService.java

```diff
@@ -0,0 +1,84 @@
+package services;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ThreadLocalRandom;
+
+import model.Board;
+import model.JumpEntity;
+import model.Player;
+import util.Constants;
+
+public class SnakeLadderService {
+	
+	public static void startGame(Board gameBoard,List<Player> playerList,List<JumpEntity> snakeList,List<JumpEntity> ladderList) {
+		Map<Integer,JumpEntity> jumpEntityMap=createJumpMap(snakeList, ladderList);
+		int totalPlayers=playerList.size();
+		while(playerList.size()!=0&&playerList.size()!=totalPlayers-1) {
+			for(int playerIterator=0;playerIterator<playerList.size();playerIterator++) {
+				takeTurn(playerIterator, jumpEntityMap, gameBoard, playerList);
+			}
+		}
+	}
+	
+	private static Map<Integer,JumpEntity> createJumpMap(List<JumpEntity> snakeList,List<JumpEntity> ladderList){
```

Comment:
return instead of map of start with object Use map of start and end like Map<Integer, Integer> startVsEnd.